tests/golos_tester.hpp

```diff
@@ -25,6 +26,11 @@
         auto filtered = ::eosio::testing::filter_fields(a.get_object(), b.get_object()); \
         BOOST_CHECK_EQUAL_COLLECTIONS(a.get_object().begin(), a.get_object().end(), filtered.begin(), filtered.end()); }}
 
+#define CHECK_EQUAL_WITH_DELTA(left, right) { \
```

Comment:
Delta should be argument for macros. In this case macros can be used in more cases. In another place different DELTA may be need.